### PR TITLE
Update - Hand and palm tflite mode.

### PR DESCRIPTION
**Pull request by st-duymai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ Downloading, extraction and placement in assets folder has been managed
  automatically by `download.gradle`.
 
 If you explicitly want to download the model, you can download it from here:
-* [Palm detection model](https://github.com/google/mediapipe/raw/master/mediapipe/modules/palm_detection/palm_detection.tflite)
-* [Hand landmark detection model](https://github.com/google/mediapipe/raw/master/mediapipe/modules/hand_landmark/hand_landmark.tflite) 
+* [Palm detection model](https://raw.githubusercontent.com/google/mediapipe/350fbb2100ad531bc110b93aaea23d96af5a5064/mediapipe/modules/palm_detection/palm_detection.tflite)
```
Comment (st-daonguyen) [inline README.md:73]:
```suggestion
* [Palm detection model](https://github.com/google/mediapipe/raw/0.8.3/mediapipe/modules/palm_detection/palm_detection.tflite)
```

File: README.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ Downloading, extraction and placement in assets folder has been managed
...
+* [Hand landmark detection model](https://raw.githubusercontent.com/google/mediapipe/350fbb2100ad531bc110b93aaea23d96af5a5064/mediapipe/modules/hand_landmark/hand_landmark.tflite)
```
Comment (st-daonguyen) [inline README.md:74]:
```suggestion
* [Hand landmark detection model](https://github.com/google/mediapipe/raw/0.8.3/mediapipe/modules/hand_landmark/hand_landmark.tflite) 
```

File: app/download_model.gradle [hunk 3]
```diff
@@ -1,11 +1,11 @@
 task downloadPalmDetectionModelFile(type: Download) {
-    src 'https://github.com/google/mediapipe/raw/master/mediapipe/modules/palm_detection/palm_detection.tflite'
+    src 'https://raw.githubusercontent.com/google/mediapipe/350fbb2100ad531bc110b93aaea23d96af5a5064/mediapipe/modules/palm_detection/palm_detection.tflite'
     dest project.ext.ASSET_DIR + '/palm_detection.tflite'
     overwrite false
 }
 
 task downloadHandDetectionModelFile(type: Download) {
-    src 'https://github.com/google/mediapipe/raw/master/mediapipe/modules/hand_landmark/hand_landmark.tflite'
+    src 'https://raw.githubusercontent.com/google/mediapipe/350fbb2100ad531bc110b93aaea23d96af5a5064/mediapipe/modules/hand_landmark/hand_landmark.tflite'
```
Comment (st-daonguyen) [inline app/download_model.gradle:8]:
```suggestion
    src 'https://github.com/google/mediapipe/raw/0.8.3/mediapipe/modules/hand_landmark/hand_landmark.tflite'
```

File: app/download_model.gradle [hunk 4]
```diff
@@ -1,11 +1,11 @@
 task downloadPalmDetectionModelFile(type: Download) {
-    src 'https://github.com/google/mediapipe/raw/master/mediapipe/modules/palm_detection/palm_detection.tflite'
+    src 'https://raw.githubusercontent.com/google/mediapipe/350fbb2100ad531bc110b93aaea23d96af5a5064/mediapipe/modules/palm_detection/palm_detection.tflite'
```
Comment (st-daonguyen) [inline app/download_model.gradle:2]:
```suggestion
    src 'https://github.com/google/mediapipe/raw/0.8.3/mediapipe/modules/palm_detection/palm_detection.tflite'
```

**Review by st-daonguyen — CHANGES_REQUESTED** [review]
@st-duymai Please check again, a version is better then commit hash.

**Comment by st-duymai** [comment]
I updated the urls. Please review it.

**Review by st-daonguyen — APPROVED** [review]

**Pull request merged**